src/reporting/ReputationToken.sol

```diff
@@ -11,19 +11,17 @@ import 'ROOT/reporting/Interfaces.sol';
 contract ReputationToken is DelegationTarget, Typed, Initializable, StandardToken, IReputationToken {
     using SafeMath for uint256;
 
-    string public name;
-    string public symbol;
-    uint256 public decimals;
+    // Delegated constracts cannot use string values, so we use bytes32 values here
+    bytes32 constant public name = "Reputation";
+    bytes32 constant public symbol = "REP";
```

Comment:
Regarding `constant public` a very naive experiment in Remix suggests it is not redundant.

```
pragma solidity ^0.4.13;

contract Test {
    
    string constant notDecorated = "one";
    string public constant decorated = "two";
}
```

Only presents the decorated version so there is at least some differentiation even if there is a way to access it outside of their UI.